src/main/resources/config.properties

```diff
@@ -4,7 +4,7 @@
 Builder.display.name = Run MATLAB Tests
 Builder.matlab.runner.target.file.name = runMatlabTests.m
 Builder.matlab.cobertura.support.warning = To generate a Cobertura report, use MATLAB R2017b or a newer version.
-Builder.invalid.matlab.root.error = Unable to launch MATLAB from the specified location. Verify the MATLAB root folder path.
+Builder.invalid.matlab.root.warning = Unable to find MATLAB from the specified location on this system(but perhaps it exists on some agents)
```

Comment:
This wording is awkward. Have Houman take a look. My attempt:

"Unable to determine if MATLAB exists at the specified location."

Reply:
Added @mw-hrastega